src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Microsoft.AspNetCore.Razor.LanguageServer.csproj

```diff
@@ -14,6 +14,10 @@
 
     <!-- We hide these dependencies from consumers because they're implementation details and aren't required to interact with the language server. -->
     <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="$(OmniSharpExtensionsLanguageServerPackageVersion)" PrivateAssets="All" />
+
+    <PackageReference Include="Microsoft.VisualStudio.LanguageServer.Protocol" Version="$(MicrosoftVisualStudioLanguageServerProtocolPackageVersion)" />
```

Comment:
We don't want to add VS protocol bits into our server. It'd violate the license given it's shipped in VSCode as well